kolibri/core/error_reports/test/test_models.py

```diff
@@ -0,0 +1,240 @@
+from django.test import override_settings
+from django.test import TestCase
+from django.utils import timezone
+
+from ..constants import BACKEND
+from ..constants import FRONTEND
+from ..constants import TASK
+from kolibri.core.error_reports.models import ErrorReport
+
+
+class ErrorReportsTestCase(TestCase):
```

Comment:
No blocker, but for clarity's sake,  we should probably rename this to `ErrorReportTestCase` as the tests are specific to the `ErrorReport` model and not the plugin `error_reports`.

Reply:
Will update!

Reply:
Updated!